packages/types/src/app-instance.ts

```diff
@@ -48,8 +51,8 @@ export interface SignedStateHashUpdate {
 }
 
 export interface ETHBucketAppState {
-  alice: string;
-  bob: string;
-  aliceBalance: BigNumber;
-  bobBalance: BigNumber;
+  readonly alice: string;
+  readonly bob: string;
+  readonly aliceBalance: BigNumber;
+  readonly bobBalance: BigNumber;
```

Comment:
Curious what the motivation was for this?

Reply:
Hmm I don't recall atm. Seems like it's not a strict requirement anyway so will remove.